src/nft/queries/genie/CollectionStatsFetcher.ts

```diff
@@ -57,6 +57,9 @@ export const CollectionStatsFetcher = async (addressOrName: string, recursive =
 
   const data = await r.json()
   const collections = data?.data.map((collection: Record<string, unknown>) => {
+    // @ts-ignore
+    collection.stats.floor_price = collection.floorPrice
```

Comment:
I don't want to spend a ton of time trying to fix these types since we are moving away from this pretty soon